packages/react-scripts/config/paths.js

```diff
@@ -56,6 +57,13 @@ const resolveModule = (resolveFn, filePath) => {
   return resolveFn(`${filePath}.js`);
 };
 
+let cachePath = resolveApp('node_modules/.cache');
```

Comment:
I see the comment you linked to (https://github.com/facebook/create-react-app/pull/9977#discussion_r515618643), but i'm trying to understand the case for when this folder wouldn't be writable? We have a few of other packages (terser, babel) that write to this location already.